astore/astore_test/e2e_test.go

```diff
@@ -0,0 +1,77 @@
+package astore_test
```

Comment:
ah ... generally have a preference for keeping the tests in the same directory as the code, right next to the code, so it's more visible and evident to random developer that there is a test, and where the test is.

Can you move up directly in astore?